storm-server/src/main/java/org/apache/storm/scheduler/Topologies.java

```diff
@@ -76,6 +76,14 @@ public Topologies(Topologies src) {
         return ret;
     }
 
+    public void addTopology(TopologyDetails details) {
```

Comment:
This is supposed to be immutable class. just create another instance of topologies using constructor.

Reply:
We shouldn't add methods to regular code while it's only used in unit test